charts/portal/templates/cronjob-backend-processes.yaml

```diff
@@ -147,6 +147,8 @@ spec:
               value: "{{ .Values.backend.useDimWallet }}"
             - name: "APPLICATIONCHECKLIST__BPDM__STARTSHARINGSTATEASREADY"
               value: "{{ .Values.backend.processesworker.bpdm.startSharingStateAsReady }}"
+            - name: "APPLICATIONCHECKLIST__BPDM__BUSINESSPARTNERPOOLBASEADDRESS"
+              value: "{{ .Values.bpdm.poolAddress }}{{ .Values.bpdm.poolApiPath }}"
```

Comment:
this is referring to a path:
https://github.com/eclipse-tractusx/portal/blob/40790029c29f97984f49159fbe746c59390f0689/environments/helm-values/values-int.yaml#L28
but expected is a full url:
https://github.com/eclipse-tractusx/portal-backend/blob/c649d4d243c4d84d1b840d5b1d101ac77750993d/src/externalsystems/Bpdm.Library/BpdmService.cs#L188

Reply:
Config param `APPLICATIONCHECKLIST__BPDM__BUSINESSPARTNERPOOLBASEADDRESS` will have the following value:

`https://business-partners-pool.example.com/pool/v6`

and rest of the url part (`v6/cx-memberships`) has already been placed while calling it from httpClient:
https://github.com/eclipse-tractusx/portal-backend/blob/c649d4d243c4d84d1b840d5b1d101ac77750993d/src/externalsystems/Bpdm.Library/BpdmService.cs#L195C42-L195C59

So, it will be: 
`https://business-partners-pool.example.com/pool/v6/v6/cx-memberships`

So, I think no full url is expected.

However, since url part (`v6/cx-memberships`) has already been placed so, I can remove the `poolApiPath` from `value` of config param `APPLICATIONCHECKLIST__BPDM__BUSINESSPARTNERPOOLBASEADDRESS`

So, then config param `APPLICATIONCHECKLIST__BPDM__BUSINESSPARTNERPOOLBASEADDRESS` will be:

`https://business-partners-pool.example.com/`

and rest of the url part (`v6/cx-memberships`) will make it:

`https://business-partners-pool.example.com/v6/cx-memberships`